mewarpx/mewarpx/utils_store/profileparser.py

```diff
@@ -0,0 +1,160 @@
+from os import write
+import sys
+import re
+import json
+
+
+class FullProfile(object):
+    """
+    Class for parsing, storing and writing the full profile data
+    """
+    def __init__(self, lines, stdout_path, write_dir):
+        self.lines = lines
+        self.stdout_path = stdout_path
+        self.write_dir = write_dir
+
+        if not self.write_dir:
+            if "/" in self.stdout_path:
+                self.write_dir = self.stdout_path[0 : self.stdout_path.rindex("/")] + "/"
+            else:
+                self.write_dir = "./"
+        elif "/" not in self.write_dir:
+            self.write_dir += "/"
```

Comment:
This can be removed if you use `os.path.join` whenever you are concatenating strings into a path.